repository/src/main/java/org/apache/atlas/GraphTransactionInterceptor.java

```diff
@@ -128,7 +128,11 @@ public Object invoke(MethodInvocation invocation) throws Throwable {
                 } else {
                     doRollback(logRollback, t);
                 }
-                throw t;
+                if (checkForBatchTooLargeError(t)) {
```

Comment:
I guess these changes have come from the previous PR.

Reply:
yes.